### PR TITLE
Feature/#217 contains duplicate

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

**Review by ryujumoto — COMMENTED** [review]
Can you solve this with time complexity O(n)?

**Comment by Rwatana** [comment]
I couldn't come up with an idea that time complexity is O(n).
Please tell me.
@Ryuju0911

File: 217_contains_duplicate.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# time: O(1)
+# space: O(1)
```
Comment (ryujumoto) [inline 217_contains_duplicate.py]: Time and space are different.

File: 217_contains_duplicate.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+from typing import List
+
+
+class Solution:
+    def containsDuplicate(self, nums: List[int]) -> bool:
+        if len(nums) == len(set(nums)):
+            return False
+        return True
```
Comment (ryujumoto) [inline 217_contains_duplicate.py]:
```
return len(nums) != len(set(nums))
```

**Comment by ryujumoto** [comment]
The solution you came up with is O(n), so it's fine.

**Comment by Rwatana** [comment]
I revised Please check my code.

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**